File: practice_java_program/notpad/RevStringWithoutChangPosition.java
public class RevStringWithoutChangPosition{
	public static void main(String[] args){
		String s = "this is my pen";  //o/p- nep ym si siht
		char[] ch=s.toCharArray();
		int left = 0;
		int right = ch.length-1;
		while(left<right){
			char temp=ch[left];
			ch[left]=ch[right];
			ch[right]=temp;

			left++;
			right--;
		}
		System.out.println(new String(ch));
	}
}

File: python_oriented_interview/notpad/RevStringWithoutChangPosition.py
s = "this is my pen" # o/p- nep ym si siht
ch = list(s)
left = 0
right = len(ch) - 1
while left < right:
    temp = ch[left]
    ch[left] = ch[right]
    ch[right] = temp
    left += 1
    right -= 1

print("".join(ch))
